packages/sui-ssr/server/utils/factory.js

```diff
@@ -28,6 +29,13 @@ export default ({path, fs, config: ssrConf = {}, assetsManifest}) => {
     return site.includes(publicFolderPrefix) ? site : `${publicFolderPrefix}${site}`
   }
 
+  const multiSiteStaticsFolder = site => {
+    const staticsFolderPrefix = `${DEFAULT_STATICS_FOLDER}-`
+    // Keep compatibility with those multi site configurations
+    // that already define the statics folder.
+    return site.includes(staticsFolderPrefix) ? site : `${staticsFolderPrefix}${site}`
+  }
+
```

Comment:
The original method was created because the `multisiteConfig` could contain a value with only the site (`dogs`) or also the containing the public folder prefix (`public-dogs`), so this method should only retrieve the actual site:

```suggestion
  const multiSiteStaticsFolder = siteValue => {
    const publicFolderPrefix = `${DEFAULT_PUBLIC_FOLDER}-`
    // Keep compatibility with those multi site configurations
    // that already define the public folder.
    const site = siteValue.replace(publicFolderPrefix, '')
 
    return `${DEFAULT_STATICS_FOLDER}-${site}`
  }

```